tests/transactional/disable_selinux.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: transactional-update
+# Summary: Disable SELinux on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+
+
+
+
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+use transactional qw(process_reboot);
+use version_utils qw(is_leap_micro is_microos is_sle_micro is_public_cloud);
+use Utils::Systemd qw(systemctl);
+use Utils::Logging 'save_and_upload_log';
+use transactional qw(trup_call check_reboot_changes process_reboot);
+use publiccloud::utils qw(allow_openqa_port_selinux);
+use bootloader_setup 'replace_grub_cmdline_settings';
+use selinuxtest 'check_disabled';
+
+
+sub run {
+    select_console 'root-console';
+
+    # disable SELinux
+    record_info('Disable', "Disable SELinux");
+    assert_script_run "sed -i -e 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config";
+    # See "Note: Relabeling your system after switching from the disabled mode" section on:
+    # https://documentation.suse.com/sle-micro/5.4/html/SLE-Micro-all/cha-selinux-slemicro.html
+    assert_script_run "touch /etc/selinux/.autorelabel";
+    # DEPRECATED runtime disable
+    # see https://github.com/SELinuxProject/selinux-kernel/wiki/DEPRECATE-runtime-disable
+    replace_grub_cmdline_settings('security=selinux selinux=1', 'security=selinux selinux=0', update_grub => 1);
```

Comment:
I'd split this into two `replace_grub_cmdline_settings` commands because we cannot be sure that the order of those kernel parameter will stay like this.

Or perhaps tust remove the `security=selinux`:

```suggestion
        replace_grub_cmdline_settings('selinux=1', 'selinux=0', update_grub => 1);
```

Reply:
Done.